tools/cloudhsm-cleanup/delete_keys_by_handle.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e
+
+CLOUDHSM_USER=${CLOUDHSM_USER:?}
+CLOUDHSM_PIN=${CLOUDHSM_PIN:?}
```

Comment:
I think we call it pin the autograph config, but I'll rename it to be consistent with the AWS docs. These scripts fall more under more general CloudHSM management than autograph specific stuff, so that'll be less confusing.

Reply:
Fixed in bbf35c7